webapp/loggedin/modules/users/components/userGroupDropdown.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
```

Comment:
I'd rather call this DropdownUserGroup. what do you think?

Reply:
ok